feature.env

```diff
@@ -25,5 +25,5 @@ MANAGE_A_WARRANT_FOLDER_UI_URL=http://localhost:9091/manageawarrant
 PATHFINDER_UI_URL=http://localhost:9091/pathfinderui
 MANAGE_SOC_CASES_UI_URL=http://localhost:9091/managesoccasesui
 WELCOME_PEOPLE_INTO_PRISON_UI_URL=https://welcome-dev.prison.service.justice.gov.uk
-FEEDBACK_ENABLED_PRISONS=LEI
+FEEDBACK_ENABLED_PRISONS=
```

Comment:
As per other comment, could leave this in to enable the test. Maybe change the prison code to a non-real like `YYY` one to avoid potential confusion?